R/musMusculus/GSE124952/GSE124952.r

```diff
@@ -23,7 +23,7 @@ meta_PFC
 sum(colnames(mat_PFC) %in% meta_PFC$CellType)
 ncol(mat_PFC)
 
-source("~/Reference-Matrix-Generation/R/utils/check.r")
+source("~/Reference-Matrix-Generation/R/utils/utils.r")
```

Comment:
switch to using `here()` before merging.

Reply:
See commit 09b05f9